interview-questions.md

```diff
@@ -8,21 +8,21 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the virtual DOM in React?
 
-Your answer:
+Your answer: its a copy of the actual DOM generated by REACT
```

Comment:
When answering interview questions, try to expand whenever possible.  One-liner sentences do not stand out from the crowd in an interview.  For this reason, work on providing more content or examples to support your answer.